architecture/core_objects/openshift_model.adoc

```diff
@@ -514,18 +520,20 @@ Sample Identity:
 }
 ----
 
-<1> Identity name must be in the form providerName:providerUserName
-<2> The name of the identity provider
```

Comment:
Checked into this, and yeah, needed further edits. I changed the instances around and it should be both correctly worded and punctuated.